tools/cmake/tests/mbed_test_mode/CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) 2020 ARM Limited. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.19.0 FATAL_ERROR)
+
+set(MBED_PATH ${CMAKE_CURRENT_SOURCE_DIR}/../../../.. CACHE INTERNAL "")
+set(MBED_CONFIG_PATH ${CMAKE_CURRENT_BINARY_DIR} CACHE STATIC "")
+
+include(${MBED_PATH}/tools/cmake/app.cmake)
+
+add_executable(mbed-test-mode-check)
+project(mbed-test-mode-check)
+
+set(PROJECT_NAME ${CMAKE_PROJECT_NAME} CACHE INTERNAL "")
+include(CTest)
```

Comment:
I mean, for a typical/existing Greentea test, we are not including CTest because we currently don't need any CTest features. Unless we use that for automation in the future?

Reply:
I believe we will use that for automation.

Reply:
Cool, thanks